includes/parse_md.php

```diff
@@ -8,8 +8,9 @@
 require_once('functions.php');
 
 // Markdown parsing libraries
-require_once(dirname(__FILE__).'/libraries/parsedown/Parsedown.php');
-require_once(dirname(__FILE__).'/libraries/parsedown-extra/ParsedownExtra.php');
+require "../vendor/autoload.php";
```

Comment:
Do you need this again as it's already been done in `functions.php`?

Reply:
thanks, I fixed it.